tasks/jnv.unattended-updates.yml

```diff
@@ -12,6 +12,8 @@
 - name: get list of nginx packages
   shell: "apt-mark showauto | grep nginx"
```

Comment:
we might be able to re-write what's there like:

```python
shell: "apt-mark showauto"

then this (maybe?)

```bash
with_items: `packages.stdout_lines.find('nginx')`
```